ci-operator/step-registry/openshift/e2e/ovirt/conformance/release-4.4-4.5/openshift-e2e-ovirt-conformance-release-4.4-4.5-workflow.yaml

```diff
@@ -1,5 +1,5 @@
 workflow:
-  as: openshift-e2e-ovirt-conformance-without-csi
```

Comment:
Why is it renamed? `no-csi` explains the purpose of this workflow, as it may be used in other releases too

Reply:
That is what I thought in the beginning, but now we will need a specific workflow for 4.6/4.7 due to added features to the installer and that will just get more and more confusing...
We agreed that locking the workflow to the release and explaining in the description is more helpful for us

Reply:
>we will need a specific workflow for 4.6/4.7

how many workflows do we need and why these can't be split into csi/no csi?

Reply:
We will need workflows for:
4.5 and 4.4 - no CSI
4.6 and 4.7 - no affinity groups
4.8 and above.

Its not critical but we just think it is simpler to pin it to the release name since it will not be backported